app/[locale]/calibrate/page.tsx

```diff
@@ -309,6 +312,12 @@ export default function Page() {
     }
   }
 
+  function handlePointerUp() {
+    if (magnifying) {
+      setMagnifying(false);
```

Comment:
Is this intended to stop magnifying when I release my mouse button? I could not get this to work on the preview; the only way to get back to a working state I have found so far is to use center + reset. Not sure if this is some bug related to my laptop or if I'm misunderstanding the magnify tool.

Reply:
Oops, looks like I somehow deleted the code for this! You're understanding is correct. Just me being sloppy.